app/main/posts/views/mode-context-form-filter.directive.js

```diff
@@ -66,7 +65,11 @@ function ModeContextFormFilter($scope, FormEndpoint, PostEndpoint, $q, _, $rootS
     }
 
     function goToUnmapped() {
-        $scope.PostFilters.setFilters({unmapped: ['unmapped']});
+        var filters = PostFilters.getDefaults();
+        filters.form.push('none');
+        filters.has_location = 'unmapped';
+        PostFilters.setFilters(filters);
+        // is there way of going to list and change the currentView without hardcoding the path?
```

Comment:
No annoyingly enough there isn't, we don't have states in the way rollcall does yet.